ang/test/civicase/shared/filters/crm-url.filter.spec.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-env jasmine */
+(function (CRM, _) {
+  describe('crmUrl', function () {
+    var $filter, crmUrl;
+
+    beforeEach(module('civicase'));
+
+    beforeEach(inject(function (_$filter_) {
+      $filter = _$filter_;
+      CRM.url = jasmine.createSpy('url');
+
+      initCrmUrlFilter();
+    }));
+
+    describe('getting the href', function () {
+      var hrefLocation, expectedHref, expectedHrefLocation, expectedQuery;
+
+      describe('when url starts with backslash', () => {
+        beforeEach(function () {
+          var originalHref = '/civicrm/a/civicase';
```

Comment:
Any reason of not putting `originalHref` to the other variable declaration list (`var hrefLocation, expectedHref, expectedHrefLocation, expectedQuery;`)?

Reply:
`originalHref` is not needed outside of `beforeEach` like other variables, so it was not kept in the global list. But its fine. changed it.